### PR TITLE
Fix/use new button component

### PR DESCRIPTION
**Pull request by vanpertsch**
The default button style changed. 

Screenshot 2022-02-17 at 14 09 31
Screenshot 2022-02-17 at 14 04 32

### CODE REVIEW

File: apps/dav/src/views/Availability.vue [hunk 1]
```diff
@@ -42,11 +42,13 @@ import {
 } from '../service/CalendarService'
 import jstz from 'jstimezonedetect'
 import TimezonePicker from '@nextcloud/vue/dist/Components/TimezonePicker'
+import { Button } from '@nextcloud/vue'
 export default {
 	name: 'Availability',
 	components: {
 		CalendarAvailability,
 		TimezonePicker,
+		Button
```
Comment (CarlSchwan) [inline apps/dav/src/views/Availability.vue:51]:
```suggestion
		Button,
```

File: apps/files/src/components/TransferOwnershipDialogue.vue [hunk 2]
```diff
@@ -86,6 +87,7 @@ export default {
 	name: 'TransferOwnershipDialogue',
 	components: {
 		Multiselect,
+		Button
```
Comment (CarlSchwan) [inline apps/files/src/components/TransferOwnershipDialogue.vue:90]:
```suggestion
		Button,
```

**Review by CarlSchwan — APPROVED** [review]

**Review by PVince81 — APPROVED** [review]
:+1:

**Review by jancborchardt — APPROVED** [review]
Looks good! :)

There is some more whitespace on top and bottom, is that from the component – and should that be? @marcoambrosini

**Review by marcoambrosini — APPROVED** [review]

**Comment by marcoambrosini** [comment]
The whitespace doesn't come from the component @jancborchardt

**Comment by tcitworld** [comment]
@vanpertsch You seem to have added a lot of unrelated commits in this PR. Do you need help with cleaning that?

**Comment by vanpertsch** [comment]
@tcitworld that would be amazing. :). (i tried to rebase, solve conflicts. Than i pulled the changes, solved conflicts. ... and am a bit lost and wonder what would be the right way to do it)

File: apps/dav/src/views/Availability.vue [hunk 3]
```diff
@@ -42,11 +42,13 @@ import {
 } from '../service/CalendarService'
 import jstz from 'jstimezonedetect'
 import TimezonePicker from '@nextcloud/vue/dist/Components/TimezonePicker'
+import { Button } from '@nextcloud/vue'
```
Comment (tcitworld) [inline apps/dav/src/views/Availability.vue:45]:
```suggestion
import Button from '@nextcloud/vue/dist/Components/Button'
```

File: apps/files/src/components/TransferOwnershipDialogue.vue [hunk 4]
```diff
@@ -72,6 +72,7 @@ import { generateOcsUrl } from '@nextcloud/router'
 import { getFilePickerBuilder, showSuccess } from '@nextcloud/dialogs'
 import Multiselect from '@nextcloud/vue/dist/Components/Multiselect'
 import Vue from 'vue'
+import { Button } from '@nextcloud/vue'
```
Comment (tcitworld) [inline apps/files/src/components/TransferOwnershipDialogue.vue:75]:
```suggestion
import Button from '@nextcloud/vue/dist/Components/Button'
```

**Comment by skjnldsv** [comment]
:tada: :tada: :tada:

**Pull request merged**